src/routes/authRoutes.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express')
+const router = express.Router()
+const { register, verifyOtp, login } = require('../controllers/authController')
+
+router.post('/signup', register)
```

Comment:
Authentication endpoints must be registered under /api/auth
```suggestion
router.post('/signup', register)
```